crates/metastoreproto/src/types/options.rs

```diff
@@ -916,7 +918,7 @@ impl From<TunnelOptionsDebug> for options::TunnelOptionsDebug {
     }
 }
 
-#[derive(Debug, Clone, Arbitrary, PartialEq, Eq)]
+#[derive(Debug, Clone, Arbitrary, PartialEq, Eq, Serialize, Deserialize)]
```

Comment:
todo: remove these. Was just using them to make sure everything was properly serializable.